src/main/java/com/microfocus/application/automation/tools/octane/tests/junit/JUnitExtension.java

```diff
@@ -229,7 +230,9 @@ public GetJUnitTestResults(Run<?, ?> build, HPRunnerType hpRunnerType, List<File
 			if (HPRunnerType.UFT.equals(hpRunnerType) || HPRunnerType.UFT_MBT.equals(hpRunnerType)) {
 
 				//extract folder names for created tests
-				String reportFolder = buildRootDir + "/archive/UFTReport";
+				String reportFolder = buildRootDir + "/archive/UFTReport" +
```

Comment:
Hi Nissim, if possible I would suggest to declare a variable (i.e. String currentNode = JenkinsUtils.getCurrentNode(workspace) ) and use it instead of calling the getCurrentNode() three times. What do you think?